### ISSUE TITLE
Preferences, Extensions, Repository dropdown popup - align content left

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#4360]
The whole dropdown menu is still Blender layout. We align the checkboxes and the content left.

![extensions](https://github.com/Bforartists/Bforartists/assets/13919702/e9bd9341-eb04-4f69-ba18-a7c390826791)

**Comment by ReinerBforartists** [comment]
Documenting

![panel](https://github.com/Bforartists/Bforartists/assets/13919702/1f437971-24e3-4333-9c0d-7bc986d586cb)

**Comment by ReinerBforartists** [comment]
U kidding me, right? :D

The whole chapter still needs a rewrite. So let's call it a day.

**Comment by Draise14** [comment]
Yeah, this chapter will be a big one, but after they stop doing changes and we finish ours, let's do the chapter. Thanks for the good indents!
